test/alternator/test_number.py

```diff
@@ -25,20 +25,18 @@
 # is a number). The tests in this file focus just on the precision and
 # magnitude that the number type can store.
 
-import decimal
+import pytest
 from decimal import Decimal
+from botocore.exceptions import ClientError
+from test.alternator.util import random_string, client_no_transform
```

Comment:
It is the last import...

The two lines below aren't a regular "import", the code of the test doesn't need to use boto3.dynamodb.types or decimal. They are part of the monkey patching, and deliberately after the comment. I don't understand why we need a bible^H^H^H^H^HPEP8 that tries to make this "bad". It's not bad. I think it makes it very clear what this workaround is doing.

Reply:
It's not the last. You can easily move it after monkey patch piece of code separating them with one empty line, so it will be clear where the monkey patch is over.

Reply:
It is the last **intended** import.
Yes, we have two further lines with the word "import" but they are not meant to be import. If I could, I wouldn't even do those imports (maybe I should have imported them inside a nested function or something).

Yes, I can put the "monkey patch" hack in the middle of the block of imports and it will make PEP8 happy. But the way I did it is better. It puts the import block at the top, and the monkey patch hack (that will hopefully go away in 10 years when the boto3 guys fix their bug) below it, separated by blank lines and a comment.